Automattic/wp-calypso
# Add instructions step for Jetpack manual activation flow

#### Changes proposed in this Pull Request

* Add the second step (instructions) of the post-purchase Jetpack manual activation flow.
* Create the `<LicensingActivation />` component which can be reused in both the manual and automatic activation flows.

**Notes**
- ~~The image on the right panel is not the correct one. I'll replace it in a follow-up PR. This will require updating the `<LicensingActivation />` component to accept the background image as a prop.~~
- We will have to revisit the styles for headers, paragraphs, and other elements to make sure we're following our designs as closely as possible. We can address this in a follow-up PR. The idea would be to add these generic styles to `<LicensingActivation />` rather than for each individual step.
- We need to clean `client/my-sites/checkout/checkout-thank-you/style.scss` since many of those styles are part of  `<LicensingActivation />`. We should only add things to that file that are unique to a step and that do not apply to all steps (e.g., the style of the header or the logo at the top). We can deal with this in a follow-up PR.
- We need to apply the `<LicensingActivation />` to the steps of the automatic activation flow (the one with the dropdown). We can do this in a follow-up PR as well.

#### Testing instructions

* Download this PR.
* Start Calypso with `yarn start`.
* Visit `http://calypso.localhost:3000/checkout/jetpack/thank-you/licensing-auto-activate/jetpack_backup_daily`.
* Select the `I don't see my site. Let me configure it manually` option from the dropdown.
* Make sure you are redirected to `http://calypso.localhost:3000/checkout/jetpack/thank-you/licensing-manual-activate/jetpack_backup_daily`. See captures at the bottom for a visual reference.
* Click on `Continue`.
* Make sure you are redirected to `http://calypso.localhost:3000/checkout/jetpack/thank-you/licensing-manual-activate-instructions/jetpack_backup_daily`. See captures at the bottom for a visual reference.

Related to 1201096622142517-as-1201096622142541

#### Captures
![image](https://user-images.githubusercontent.com/3418513/140328781-9886e52a-565e-4f45-b17b-9b4097a47322.png)

![image](https://user-images.githubusercontent.com/3418513/140412050-5424b9fa-7e6c-40c9-8f19-5afaaf20a9a3.png)
